packages/react/src/hooks/useIonPicker.tsx

```diff
@@ -9,9 +9,10 @@ import { useController } from './useController';
 /**
  * A hook for presenting/dismissing an IonPicker component
  * @returns Returns the present and dismiss methods in an array
+ * @deprecated Use the inline ion-picker component instead.
  */
 export function useIonPicker(): UseIonPickerResult {
-  const controller = useController<PickerOptions, HTMLIonPickerElement>(
+  const controller = useController<PickerOptions, HTMLIonPickerLegacyElement>(
     'IonPicker',
```

Comment:
Just to confirm my understanding of why we don't rename 'IonPicker' here in line 16: We don't need to rename this because it's used for the hook & controller, which we in turn don't need to rename because there's no equivalent in IonPickerInternal. Do I have that right?

Reply:
Correct!